package.json

```diff
@@ -26,7 +26,8 @@
 	"private": true,
 	"workspaces": [
 		"src/core/*",
-		"src/core/components/*"
+		"src/core/components/*",
+		"src/editorial/web/components/*"
```

Comment:
Why does `editorial` have a `web` subset but `core` doesn't?

Reply:
No real reason, just haven't got around to it 😄